### DISCUSSION TITLE
Tera wallet plugin

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5055] [Working on Features]
"Wcfm also says there plugin is compatible with tera wallet plugin i suggest you to add this feature in next release because tera wallet is both for vendor and customer"

**Comment by HarrySmith01** [comment]
This is something we should already have as Wallet is one of common feature for an app or website.

**Comment by Mokwa123** [comment]
Please add tera wallet on next release

**Comment by tjain90** [comment]
Yes please add this. This is one of the best wallet feature for woocomerce.

**Comment by sirlojik** [comment]
yes, Terrawallet is essential esp with refunding

**Comment by JohnTan86** [comment]
Must have feature

**Comment by freefugga** [comment]
This is a must have feature. Please add TeraWallet Support with new release asap. Thanks

**Comment by ameenbadri7** [comment]
If fluxstore add TeraWallet that will take it to the next level, customers can use it as a local payment option for country like my country.
https://wordpress.org/plugins/woo-wallet/

TeraWallet allows customers to store their money in a digital wallet. The customers can use the wallet money for purchasing products from the store. The customers can add money to their wallet using various payment methods set by the admin. The admin can set cashback rules according to cart price or product. The customers will receive their cashback amount in their wallet account. The admin can process refund to customer wallet.

**Comment by thesaluja** [comment]
Yes, please integrate terawallet along with notifications of credit/debit

**Comment by ekramul-1** [comment]
Please add tera wallet as this is the very essential thing.

**Comment by Ekesmall** [comment]
Please add Tera wallet as soon as you can

**Comment by abedfatafta** [comment]
helpfull , we need this

**Comment by ruwad-co** [comment]
please add wallet system

**Comment by 8ractmedia** [comment]
Please Add wallet system

**Comment by Zunket** [comment]
This will be much beneficial.

**Comment by krrisharora** [comment]
Yes Tera Wallet must be added in the next version

**Comment by QuickBasket** [comment]
It is not a nice to have but a basic necessity. Must have feature.

**Comment by aliadnandeveloper** [comment]
Yes, I do recommend adding Tera Wallet. Thank You!

**Comment by minhcasi** [comment]
Hi all, just update that we are working on this feature 👍

**Reply by Bhatsha** [reply]
what is the release date for Tera wallet plugin integrated version

**Reply by shubhamrane48** [reply]
hello, can you please give any expected release date with tera wallet integration.

**Comment by Ekesmall** [comment]
We're waiting eagerly for this feature, please 🙏 add it soon

**Comment by Bhatsha** [comment]
what is the release date for Tera wallet plugin integrated version

**Comment by shakirkedir** [comment]
Please add tera wallet

**Comment by antimimin** [comment]
But free tera wallet cannot withdraw.

**Comment by QuickBasket** [comment]
Terawallet seems to be in code now and initial testing is encouraging. Thanks.

**Comment by pivotmobileapp** [comment]
When EnableOnePageCheckout is set to true... TopUp wallet doesnt work

**Reply by ecomdeveloperbd** [reply]
I am also facing many problems. Not working properly

**Comment by ecomdeveloperbd** [comment]
TerraWallet button for transfer and deposit isn’t working.
Many problem 2.5.1 version. Please fix

**Comment by hassanyousufx** [comment]
TeraWallet Top Up. Delivery fee is being added when doing wallet top-up.  Please look

**Comment by hassanyousufx** [comment]
Top up is not working
